src/docs/user/references/Variables.adoc

```diff
@@ -145,9 +145,12 @@ Examples: PA:MODEL_FROM_URL(file:///srv/machines_list_model.txt), if the file ma
 * *PA:CRON* : variable syntax must be a valid cron expression as defined in the  http://www.sauronsoftware.it/projects/cron4j/manual.php#p02[cron4j manual]. +
 Examples: PA:CRON will accept "5 * * * \*" but not "* * * *" (missing minutes sub-pattern).
 * *PA:SPEL(SpEL expression)* : variable syntax will be evaluated by a https://docs.spring.io/spring/docs/current/spring-framework-reference/html/expressions.html[SpEL] expression. Refer to next paragraph.
-* *PA:CATALOG_OBJECT* : variable syntax must be a valid expression that matches the following pattern: `bucketName/objectName[/revision]`.
+* *PA:CATALOG_OBJECT*, *PA:CATALOG_OBJECT(kind)*, *PA:CATALOG_OBJECT(,contentType)*, *PA:CATALOG_OBJECT(kind,contentType)* : variable syntax must be a valid expression that matches the following pattern: `bucketName/objectName[/revision]`.
 Note that the revision sub-pattern is a hash code number represented by 13 digit. +
+A filter can be specified in the PA:CATALOG_OBJECT model. In that case, the variable value must be a catalog object which matches the kind and content type requirements.
+The scheduler server verifies that the object exists in the catalog and fulfills the requirements during workflow submission. +
```

Comment:
"during workflow submission..."
is the check done even continuously before clicking on the check button?

Reply:
No, actually I meant that before we click the check or submit button, it's not checked (e.g., during workflow design). I've changed it to "when the workflow is submitted". Thank you.